src/codegen/mod.rs

```diff
@@ -1832,16 +1832,12 @@ impl From<&ast::Builtin> for Builtin {
     }
 }
 
-pub(super) fn error_msg_with_loc(ns: &Namespace, error: &str, loc: Option<Loc>) -> String {
-    if let Some(loc) = loc {
-        match loc {
-            Loc::File(..) => {
-                let loc_from_file = ns.loc_to_string(PathDisplay::Filename, &loc);
-                format!("runtime_error: {error} in {loc_from_file}")
-            }
-            _ => error.to_string(),
+pub(super) fn error_msg_with_loc(ns: &Namespace, error: String, loc: Option<Loc>) -> String {
```

Comment:
Is it really worth it to change this from `&str` to `String`?

Reply:
`fn log_runtime_error` is the main function that calls `error_msg_with_loc`. It was receiving a `String` as the argument, that was converted to `&str` for `error_msg_with_loc`, which would convert again `&str` to `String`. This scheme meant two allocations for the same string.